src/backend/src/entities/game.entity.ts

```diff
@@ -5,7 +5,6 @@ import {
   OneToMany,
   BaseEntity,
 } from 'typeorm';
-import { Grab } from './grab.entity';
 import { Player } from './player.entity';
```

Comment:
is there a reason for this delete at all? was it unused

Reply:
Yeah, we didn't use it